content-src/components/DiscoveryStreamBase/DiscoveryStreamBase.jsx

```diff
@@ -2,24 +2,13 @@ import {CardGrid} from "content-src/components/DiscoveryStreamComponents/CardGri
 import {connect} from "react-redux";
 import {Hero} from "content-src/components/DiscoveryStreamComponents/Hero/Hero";
 import {HorizontalRule} from "content-src/components/DiscoveryStreamComponents/HorizontalRule/HorizontalRule";
-import {ImpressionStats} from "content-src/components/DiscoveryStreamImpressionStats/ImpressionStats";
 import {List} from "content-src/components/DiscoveryStreamComponents/List/List";
 import {Navigation} from "content-src/components/DiscoveryStreamComponents/Navigation/Navigation";
 import React from "react";
 import {SectionTitle} from "content-src/components/DiscoveryStreamComponents/SectionTitle/SectionTitle";
 import {selectLayoutRender} from "content-src/lib/selectLayoutRender";
 import {TopSites} from "content-src/components/DiscoveryStreamComponents/TopSites/TopSites";
 
-// According to the Pocket API endpoint specs, `component.properties.items` is a required property with following values:
-//   - List 1-12 items
-//   - Hero 1-5 items
-//   - CardGrid 1-16 items
-// To enforce that, we define various maximium items for individual components as an extra check.
-// Note that these values are subject to the future changes of the specs.
-const MAX_ROWS_HERO = 5;
-const MAX_ROWS_LIST = 12;
-const MAX_ROWS_CARDGRID = 16;
```

Comment:
Is this something we're not enforcing anymore? I don't see it in the patch anymore.

Reply:
Ah, I see, because we're now wrapping the contents inside the components, and not around the outside of the components, we have access to the content already that these consts were being used to get.